library/alloc/src/alloc.rs

```diff
@@ -323,6 +321,16 @@ pub(crate) unsafe fn box_free<T: ?Sized>(ptr: Unique<T>) {
     }
 }
 
+// # Allocation error handler
+
+extern "Rust" {
+    // This is the magic symbol to call the global alloc error handler.  rustc generates
+    // it to call `__rg_oom` if there is a `#[global_allocator]`, or uses the
+    // default implementations below (`__rdl_oom`) otherwise.
+    #[rustc_allocator_nounwind]
```

Comment:
This attribute leaves me rather uneasy, since the default implementation *does* unwind... see https://github.com/rust-lang/rust/pull/76448/files#r502291279.

Reply:
If it is safe, please remove... It was blindly copied.

Reply:
I have no idea if it is safe or what else could happen. Presumably it will lead to unwinding edges on OOM calls which could have perf impact.

Reply:
EDIT (accidently removed): If it is safe, please remove... It was blindly copied.

Reply:
I think it's fine to leave the attribute for now.